internal/oci/container.go

```diff
@@ -72,7 +72,7 @@ type Container struct {
 	stdinOnce          bool
 	created            bool
 	spoofed            bool
-	stopping           bool
+	timesStopped       int
```

Comment:
I don't understand the reason to have a refcounter here, can you please explain?